x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -442,7 +462,13 @@ export const getAboutContinueButton = () => cy.get(ABOUT_CONTINUE_BTN);
 export const getDefineContinueButton = () => cy.get(DEFINE_CONTINUE_BUTTON);
 
 /** Returns the indicator index pattern */
-export const getIndicatorIndex = () => cy.get(THREAT_MATCH_INDICATOR_INDEX).eq(0);
+export const getIndicatorIndex = () => {
+  // cy.get('[id="indexPatterns"]').click({ force: true });
+  // cy.get('[data-test-subj="detectionEngineStepDefineRuleIndexPatternsAccordion"]').click({
```

Comment:
Should we delete dead code here?

Reply:
resolved here: 56bed04c108fc562752a4d34785fbd40702af678